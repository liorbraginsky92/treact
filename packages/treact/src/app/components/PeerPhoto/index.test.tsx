jest.mock('helpers/FileManager/picStore')

import * as React from 'react'
import * as renderer from 'react-test-renderer'
import { PeerPhoto, PeerPhotoEmpty } from './'

it('PeerPhoto renders correctly', () => {
  const tree = renderer.create(
    <PeerPhoto id={1} className="" />,
  ).toJSON()

  expect(tree).toMatchSnapshot()
})

it('PeerPhotoEmpty renders correctly', () => {
  const tree = renderer.create(
    <PeerPhotoEmpty className="" />,
  ).toJSON()

  expect(tree).toMatchSnapshot()
})

