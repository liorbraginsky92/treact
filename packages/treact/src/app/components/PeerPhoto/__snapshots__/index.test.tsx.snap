// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PeerPhoto renders correctly 1`] = `
<img
  className=""
  src="test-data"
/>
`;

exports[`PeerPhotoEmpty renders correctly 1`] = `
<img
  className=""
  src="test-file-stub"
/>
`;
